tests/scripts/thread-cert/otci/otci/otci.py

```diff
@@ -0,0 +1,1865 @@
+
+    def get_child_supervision_interval(self) -> int:
+        """Get the Child Supervision Check Timeout value."""
+        return self.__parse_int(self.execute_command('childsupervision interval'))
+
+    def set_child_supervision_interval(self, val: int):
+        """Set the Child Supervision Interval value.
+        This command can only be used with FTD devices.
+        """
+        self.execute_command(f'childsupervision interval {val}')
+
+    def get_child_supervision_check_timeout(self) -> int:
+        """Get the Child Supervision Check Timeout value."""
+        return self.__parse_int(self.execute_command('childsupervision checktimeout'))
+
+    def set_child_supervision_check_timeout(self, val: int):
+        """Set the Child Supervision Check Timeout value."""
+        self.execute_command(f'childsupervision checktimeout {val}')
+
+    #
+    # Neighbor management
+    #
+
+    def get_neighbor_list(self) -> List[Rloc16]:
+        """Get a list of RLOC16 of neighbors"""
+        line = self.__parse_str(self.execute_command('neighbor list')).strip()
+        return [Rloc16(id, 16) for id in line.split()]
+
+    def get_neighbor_table(self) -> Dict[Rloc16, Dict[str, Any]]:
+        output = self.execute_command('neighbor table')
+        if len(output) < 2:
+            raise UnexpectedCommandOutput(output)
+
+        #
+        # Example output:
+        #
+        # | Role | RLOC16 | Age | Avg RSSI | Last RSSI |R|D|N| Extended MAC     |
+        # +------+--------+-----+----------+-----------+-+-+-+------------------+
+        # |   C  | 0xcc01 |  96 |      -46 |       -46 |1|1|1| 1eb9ba8a6522636b |
+        # |   R  | 0xc800 |   2 |      -29 |       -29 |1|1|1| 9a91556102c39ddb |
+        # |   R  | 0xf000 |   3 |      -28 |       -28 |1|1|1| 0ad7ed6beaa6016d |
+        # Done
+        #
+
+        table = {}
+        for line in output[2:]:
+            line = line.strip()
+            if not line:
+                continue
+
+            fields = line.split('|')
+            _, role, rloc16, age, avg_rssi, last_rssi, r, d, n, extaddr, _ = fields
+
+            is_router = role.strip() == 'R'
+            r, d, n = int(r), int(d), int(n)
+            mode = DeviceMode(f'{"r" if r else ""}{"d" if d else ""}{"n" if n else ""}')
+
+            rloc16 = Rloc16(rloc16, 16)
+
+            table[rloc16] = {
+                'is_router': is_router,
+                'rloc16': rloc16,
+                'age': int(age),
+                'avg_rssi': int(avg_rssi),
+                'last_rssi': int(last_rssi),
+                'mode': mode,
+                'extaddr': extaddr,
+            }
+
+        return table
+
+    #
+    # SED/SSED configuration
+    #
+
+    def get_poll_period(self) -> float:
+        """Get the customized data poll period of sleepy end device (milliseconds).
+        Only for Reference Device."""
+        return self.__parse_int(self.execute_command('pollperiod')) / 1000.0
```

Comment:
match the doxygen `milliseconds`
```suggestion
        return self.__parse_int(self.execute_command('pollperiod'))
```